Terminal.Gui/Core/View.cs

```diff
@@ -695,7 +695,9 @@ public virtual void RemoveAll ()
 
 			while (subviews.Count > 0) {
 				Remove (subviews [0]);
-				Remove (tabIndexes [0]);
+				if (tabIndexes.Count > 0) {
```

Comment:
Done

Reply:
Thanks. The `ColorScheme `even if I create an empty view
`var v = new (View);`
 and execute a
`v.Redraw (v.Bounds);`
never gets any error. I only gets an error if I using it in a unit test without using the `Application.Init` and `Application.Run` like is obvious, because the `Colors `are created within the `Driver`. If the errors occurs when using unit tests without using `Application.Init` and `Run `you should use them with the `FakeDriver`.
Can you give me an exemple when the error happens, please?